src/test/online-boutique/values.yaml

```diff
@@ -0,0 +1,66 @@
+ad:
+  name: adservice
+  version: v1
+  image:
+    repository: gcr.io/google-samples/microservices-demo/adservice:v0.2.3
```

Comment:
gcr.io is blocked in mainland China, maybe we could sync these images to aliyun

Reply:
https://github.com/kubesphere/image-sync-config/pull/6